BUILD_macos.md

```diff
@@ -81,6 +81,9 @@ PRs with corrections and updates are welcome!
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
 
+* Teleport uses wasm-pack, so if that's not present, we need to install it. Find the wasm install docs [here](https://rustwasm.github.io/wasm-pack/installer/).
```

Comment:
```suggestion
* Teleport uses wasm-pack, so if that's not present, we need to [install it]( https://rustwasm.github.io/wasm-pack/installer/).
```

WASM and wasm-pack are not the same thing.

Reply:
Correct, fixed